config/application.rb

```diff
@@ -12,11 +12,40 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
+class JSONAPIError < Committee::ValidationError
+  def error_body
+    {
+      errors: [
+        { status: id, detail: message }
+      ]
+    }
+  end
+
+  def render
+    [
+      status,
+      { 'Content-Type' => 'application/vnd.api+json' },
+      [JSON.generate(error_body)]
+    ]
+  end
+end
+
 module PreservationCatalog
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.1
 
+    # accept_request_filter omits OKComputer
+    accept_proc = proc { |request| request.path.start_with?('/v1') }
+    config.middleware.use Committee::Middleware::RequestValidation, schema_path: 'openapi.yml',
+                                                                    strict: true, error_class: JSONAPIError,
+                                                                    accept_request_filter: accept_proc
+    # TODO: we can uncomment this at a later date to ensure we are passing back
+    #       valid responses. Currently, uncommenting this line causes 24 spec
+    #       failures.
+    #
```

Comment:
make a ticket for this and reference it?

Reply:
https://github.com/sul-dlss/preservation_catalog/issues/1407

Reply:
Done.